server/boot/passport.js

```diff
@@ -1,7 +1,7 @@
 module.exports = function (server) {
   var router = server.loopback.Router();
   router.get('/auth/success', function (req, res, next) {
-    res.redirect(require('../config').baseUrl + '/#/passport/');
+    res.redirect(require('../config').appUrl + '/#/passport/');
```

Comment:
I remember changing this so I could log in but can't remember the exact reasons, @rogersnm